tests/braille-specs/de-g2.yaml

```diff
@@ -451,14 +460,192 @@ tests:
+  # replace ambiguous double closing quotation mark by unambiguous one
+  - - "Test Test«."
+    - "te} te})."
+  # combinations of both single opening and closing quotation mark
+  - - "Test ›Test‹."
+    - "te} '(te}')."
+  - - "Test ‹Test›"
+    - "te} '(te}')."
+    - xfail: not implemented
+  - - "›Test‹ Test."
+    - "'(te}') te}."
+  - - "‹Test› Test"
+    - "'(te}') te}."
+    - xfail: not implemented
+  # combinations of both double opening and closing quotation mark
+  - - "Test »Test«."
+    - "te} (te})."
+  - - "Test «Test»."
+    - "te} (te})."
+  - - "»Test« Test."
+    - "(te}) te}."
+  - - "«Test» Test."
+    - "(te}) te}."
+  # combinations of both single and double opening and closing mark
+  - - "›Test Test »Test«‹."
+    - "'(te} te} (te})')."
+  - - "»Test Test ›Test‹«."
+    - "(te} te} '(te}'))."
+  - - "‹Test Test «Test»›"
+    - "'(te} te} (te})')."
+    - xfail: not implemented
+  - - "«Test Test ‹Test›»"
+    - "(te} te} '(te}'))."
+    - xfail: not implemented
+  # corrections of closing quotation mark with hyphen
+  - - "›Hey!‹-Ruf"
+    - "'(hey+')-ruf"
+    - xfail: not implemented
+  - - "‹Hey!›-Ruf"
+    - "'(hey+')-ruf"
+    - xfail: not implemented
+  - - "»Hey!«-Ruf"
+    - "(hey+)-ruf"
+    - xfail: not implemented
+  - - "«Hey!»-Ruf"
+    - "(hey+)-ruf"
+    - xfail: not implemented
+  - - "widerlich -‹«"
+    - "wid7_'-'))"
+    - xfail: not implemented
+
+# Case indication
+
+  # Alone standing capital letters are marked with dots 45.
+  - - "U-Bahn"
+    - ">u-bahn"
+    - xfail: not implemented
+  - - "das große M"
+    - "d g~e >m"
+    - xfail: not implemented
+
+  # Alone standing single capital letter after apostrophe (U+0027) is marked with dots 45.
+  - - "'T."
+    - "'>t."
+    - xfail: not implemented
+  - - "'T Test."
+    - "'>t te}."
+    - xfail: not implemented
+  - - "Test 'T."
+    - "te} '>t."
+    - xfail: not implemented
+  - - "Test,'T."
+    - "te},'>t."
+    - xfail: not implemented
+  - - "Test 'T Test."
+    - "te} '>t te}."
+    - xfail: not implemented
+  - - "Test,'T Test."
+    - "te},'>t te}."
+    - xfail: not implemented
+  # same for right single quotation mark (U+2019)
+  - - "’T."
+    - "'>t."
+    - xfail: not implemented
+  - - "’T Test."
+    - "'>t te}."
+    - xfail: not implemented
+  - - "Test ’T."
+    - "te} '>t."
+    - xfail: not implemented
+  - - "Test,’T."
+    - "te},'>t."
+    - xfail: not implemented
+  - - "Test ’T Test."
+    - "te} '>t te}."
+    - xfail: not implemented
+  - - "Test,’T Test."
+    - "te},'>t te}."
+    - xfail: not implemented
+
+  # No case indication when there are abbreviations with periods.
+  - - "z.B."
+    - "z.b."
+  - - "evtl."
+    - "evtl."
+  - - "a.a.O."
+    - "a.a.o."
+
+  # Capital letters within a word are marker with dots 456. We can't use dots 46 here because this
+  # is the grade 2 abbreviation of "ck".
+  - - "SchülerInnen"
+    - "58l7!$*nc"
+    - xfail: not implemented
+  - - "iPhone"
+    - "'i!$phone"
```

Comment:
Can you elaborate? I don't understand.

Reply:
As far as I understand there are slightly different rules for abbrevs. And since we treat `iPhone` as an abbrev (which is stupid IMHO) we translate it slightly differently. According this might change.

Reply:
I see, so there can be discussion about what is an abbrev and what isn't. OK, this shouldn't be an issue, as long as we make it controllable/overridable by the user.

Could you add a test that translates iPhone as an abbreviation?